### PR TITLE
Support for pgbench pipeline

### PR DESCRIPTION
**Pull request by ekuric**
Support for pgbench pipeline

Signed-off-by: Elvir Kuric <elvirkuric@gmail.com>

### CODE REVIEW

**Comment by ekuric** [comment]
@chaitanyaenr PTAL and merge - this will provide pgbench pipeline feature.

File: pipeline-scripts/pgbench.groovy [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                        // get properties file - from test location 
+                        // in SCALE-CI there will be defined PGBENCH_SCALE_TEST_PROPERTY_FILE 
+                        // for now just keep it as is -- 
+
+                        sh "wget https://raw.githubusercontent.com/ekuric/openshift/master/postgresql/pgbench.properties"
+                        // sh "wget ${PGBENCH_SCALE_TEST_PROPERTY_FILE}"
+                        sh "cat pgbench.properties"
+			def pgbench_scale_test_properties = readProperties file: "pgbench.properties"
+                        def NAMESPACE = pgbench_scale_test_properties['NAMESPACE']
+                        def TRANSACTIONS = pgbench_scale_test_properties['TRANSACTIONS']
```
Comment (chaitanyaenr) [inline pipeline-scripts/pgbench.groovy:27]: Wrong spacing.

File: pipeline-scripts/pgbench.groovy [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                        def TEMPLATE = pgbench_scale_test_properties['TEMPLATE']
+			def VOLUME_CAPACITY = pgbench_scale_test_properties['VOLUME_CAPACITY']
+			def MEMORY_LIMIT = pgbench_scale_test_properties['MEMORY_LIMIT']
+			def ITERATIONS = pgbench_scale_test_properties['ITERATIONS']
+			def MODE = pgbench_scale_test_properties['MODE']
+			def CLIENTS = pgbench_scale_test_properties['CLIENTS']
+			def THREADS = pgbench_scale_test_properties['THREADS']
+			def SCALING = pgbench_scale_test_properties['SCALING']
+                        def PBENCHCONFIG = pgbench_scale_test_properties['PBENCHCONFIG']
```
Comment (chaitanyaenr) [inline pipeline-scripts/pgbench.groovy:36]: Wrong spacing.

File: pipeline-scripts/pgbench.groovy [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                        def STORAGECLASS = pgbench_scale_test_properties['STORAGECLASS']
+
+
+			
+                        // debug info
+			println "----------USER DEFINED OPTIONS-------------------"
+			println "-------------------------------------------------"
+			println "-------------------------------------------------"
+                        println "NAMESPACE: '${NAMESPACE}'"
+                        println "TRANSACTIONS: '${TRANSACTIONS}'"
```
Comment (chaitanyaenr) [inline pipeline-scripts/pgbench.groovy:46]: Wrong spacing.

File: pipeline-scripts/pgbench.groovy [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+                                parameters: [    
+                                                [$class: 'StringParameterValue', name: 'NAMESPACE', value: NAMESPACE ],
+                                                [$class: 'StringParameterValue', name: 'TRANSACTIONS', value: TRANSACTIONS ],
+                                                [$class: 'StringParameterValue', name: 'TEMPLATE', value: TEMPLATE ],
+                                                [$class: 'StringParameterValue', name: 'VOLUME_CAPACITY',value: VOLUME_CAPACITY ],
+                                                [$class: 'StringParameterValue', name: 'MEMORY_LIMIT', value: MEMORY_LIMIT ], 
+                                                [$class: 'StringParameterValue', name: 'ITERATIONS', value: ITERATIONS ],
+                                                [$class: 'StringParameterValue', name: 'MODE', value: MODE ],
+                                                [$class: 'StringParameterValue', name: 'CLIENTS', value: CLIENTS ],
+						[$class: 'StringParameterValue', name: 'THREADS', value: THREADS ],
```
Comment (chaitanyaenr) [inline pipeline-scripts/pgbench.groovy:76]: Wrong spacing.

**Review by chaitanyaenr — CHANGES_REQUESTED** [review]
Thanks for the PR Elko. This commit is good to go, just need the spacing corrected.

**Pull request closed without merging**